sql/monitoring/telemetry_missing_columns_v1/view.sql

```diff
@@ -0,0 +1,32 @@
+CREATE
+OR REPLACE VIEW `moz-fx-data-shared-prod.monitoring.telemetry_missing_columns_v1` AS
+SELECT
+  TIMESTAMP_TRUNC(submission_timestamp, DAY) AS day,
+  'telemetry' AS document_namespace,
+  REGEXP_EXTRACT(_TABLE_SUFFIX, r "^(.*)_v.*") AS document_type,
```

Comment:
These could be useful UDFs: `udf_extract_doctype`, `udf_extract_docversion`

Reply:
I've opened #471 for this. I don't think it's possible to deploy a udf and a view at the same time.